solenspipe/solenspipe.py

```diff
@@ -377,24 +495,38 @@ def get_noise_power(self,channel=None,beam_deconv=False):
         return ls,nells,nells_P
     
 
-    def initialize_norm(self,ch,lmin,lmax,recalculate=False,quicklens=False,label=None):
+    def initialize_norm(self,ch,lmin,lmax,recalculate=False,quicklens=False,curl=False,label=None):
```

Comment:
I think this function could be improved a bit. Firstly, when using recalculate=True, an `AssertionError` is thrown. Errors should be used when something has really gone wrong, not as a consequence of using an allowed value of an option argument. A warning would be more appropriate but even that seems a bit strong. 

Secondly, and probably more importantly, we should tidy up the return values. Unless there is a very good reason to, it's generally good to write your function so it has a single format for what it is returning, because this really simplifies any code  that is calling the function. In this case, it looks like most appropriate would be in all cases to return an array of Ls and a dictionary of A(L)s. This would also be advantageous because having more than a few return values (there are currently 9, with this change there would be 2) makes bugs more likely as it's easy to mix up their ordering, and takes more effort to make changes (e.g. to add or remove a return value).